reana_client/cli/workflow.py

```diff
@@ -825,8 +858,12 @@ def interactive_group():
 @add_access_token_options
 @click.pass_context
 def workflow_open_interactive_session(ctx, workflow, interactive_session_type,
-                                      image, access_token):
-    """Open an interactive session inside the workflow workspace."""
+                                      image, access_token):  # noqa: D301
+    """Open an interactive session inside the workspace.
```

Comment:
Add some body, e.g.

The `open` command permits to open interactive session processes on top of the workflow workspace, such as Jupyter notebooks.  This is useful to quickly inspect and analyse the produced files while the workflow is stlil running.